TeacherWorkout.Migrator/DbContextExtensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Npgsql;
+using Polly;
+
+namespace TeacherWorkout.Migrator
+{
+    public static class DbContextExtensions
+    {
+        public static async Task CreateAndMigrateAsync(this DbContext context)
+        {
+            var retryPolicy = Policy
```

Comment:
This same migrator will be used to apply DB migration in the Prod env when running with docker. If we get temporary transient faults it's preferable to retry just the migration rather than have to retry the whole deploy process. Perhaps a bit overkill at this stage, but it doesn't hurt :D